consensus_decentralization/analyze.py

```diff
@@ -21,7 +21,16 @@ def analyze(projects, aggregated_data_filename, output_dir):
     """
     logging.info('Calculating metrics on aggregated data..')
     metrics = hlp.get_metrics_config()
-    metric_names = list(metrics.keys())
+    metric_values = []
+    metric_names = []
```

Comment:
This variable is not really necessary now that we have the one defined above too. I would recommend removing this one and having `f'{key}={val}'` as the first item for the metric_values entries when there is an arg, and then accessing the names from there when needed